corehq/apps/export/views.py

```diff
@@ -342,43 +342,6 @@ def max_column_size(self):
         except TypeError:
             return 2000
 
-    def _process_filters_and_specs(self, in_data):
```

Comment:
This one definitely seems too big to put if we don't have to.

Reply:
This got quite a bit shorter, mostly due to 95641ac459fd46518ae201b84d11eecfde62d95b, so I don't think re-extracting it would be valuable anymore.